src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommand implements DriverCommand {
+    ArrayList<DriverCommand> commands;
```

Comment:
Type should indicate interface (List) not the implementation.
Should be private.